### PR TITLE
Support IPv6 BGP-triggered GUE and add some debugging statements

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

**Comment by wenovus** [comment]
@DanG100 Does the dataplane support configuring both IPv4 and IPv6 addresses? If so I can later look at merging the IPv4 and IPv6 tests into a single one.

File: bgp/debug.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package bgp
```
Comment (DanG100) [inline bgp/debug.go:1]: license header

File: sysrib/server_test.go [hunk 2]
```diff
@@ -156,6 +156,173 @@ func configureInterface(t *testing.T, intf *AddIntfAction, yclient *ygnmi.Client
 	}
 }
 
+var (
+	// v4v6Mapper maps IPv4 addresses to IPv6 addresses for testing v6.
+	v4v6Mapper = map[string]string{
```
Comment (DanG100) [inline sysrib/server_test.go:161]: can this be func instead a hardcoded map?

File: sysrib/server_test.go [hunk 3]
```diff
@@ -156,6 +156,173 @@ func configureInterface(t *testing.T, intf *AddIntfAction, yclient *ygnmi.Client
...
+				isV6:      false,
+			},
+			dstIP4: [4]byte{192, 168, 1, 42},
+		}: {
+			GUEPolicy: GUEPolicy{
+				dstPortv6: 8,
+				srcIP6:    [16]byte{42, 42, 42, 42, 42},
+				isV6:      true,
+			},
+			dstIP6: [16]byte{0x20, 0x1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0x42},
```
Comment (DanG100) [inline sysrib/server_test.go:213]: maybe a test utility to go from ip str to array might be nice

**Comment by DanG100** [comment]
> @DanG100 Does the dataplane support configuring both IPv4 and IPv6 addresses and forwarding both traffic at once? If so I can later look at merging the IPv4 and IPv6 tests into a single one.

that should be supported

File: sysrib/server_test.go (hunk 2 above)
Comment (wenovus) [reply sysrib/server_test.go:161]: `mapAddress` uses this map to convert the addresses.

File: sysrib/server_test.go (hunk 3 above)
Comment (wenovus) [reply sysrib/server_test.go:213]: There is the tradeoff of [16]byte forcing a very precise definition, so it's not yet clear that such a utility will be better. Maybe with more use cases we will see a pattern.

File: sysrib/server_test.go (hunk 2 above)
Comment (wenovus) [reply sysrib/server_test.go]: Done.

**Review by DanG100 — APPROVED** [review]

**Pull request merged**